client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -257,6 +259,9 @@ function PatternPreviewFragment( {
 						borderless
 						className="pattern-preview__title"
 						onCopy={ () => {
+							recordPatternsEvent( 'calypso_pattern_permalink_copy', {
```

Comment:
```suggestion
							recordPatternsEvent( 'calypso_pattern_library_permalink_copy', {
```
Let's stick with the same prefix for all pattern library events.